src/api/db/data/20250411000109_backfill_reporter_on_reports.rb

```diff
@@ -0,0 +1,15 @@
+# frozen_string_literal: true
+
+class BackfillReporterOnReports < ActiveRecord::Migration[7.1]
+  def up
+    Report.where(reporter_id: nil).unscoped.in_batches do |batch|
+      batch.find_each do |report|
+        report.update(reporter_id: report.user_id)
```

Comment:
This will trigger validations and callbacks.

Reply:
Overlooked that, switched to `update_columns` to avoid it